### PR TITLE
Gh 1475/owner key details of external owner keys

### PR DESCRIPTION
**Pull request by MouazAlzahabi**
Handles #1475 #1476 #1477 

still needs integration with backend requests

### CODE REVIEW

**Review by DmitryBespalov — APPROVED** [review]

**Review by sche — APPROVED** [review]

File: Multisig/Cross-layer/Analytics/TrackingEvent.swift [hunk 1]
```diff
@@ -165,4 +165,10 @@ enum TrackingEvent: String, Trackable {
     case addDelegateKeySkipped                      = "user_skip_add_delegate"
     case addDelegateKeySuccess                      = "user_success_add_delegate"
     case addDelegateKeyFailed                       = "user_failed_add_delegate"
+
+    // MARK: delete  delegate key
```
Comment (sche) [inline Multisig/Cross-layer/Analytics/TrackingEvent.swift:169]: 👍🏻 
Could you please also update our tracking document here? https://docs.google.com/spreadsheets/d/1mj9iQIhpM-Pak7lQMb1dmxhQn1QdJoJyhbc730dZNzA/edit#gid=1949479240

File: Multisig/UI/UI Library/UITableView+Cells.swift [hunk 2]
```diff
@@ -74,4 +80,19 @@ extension UITableView {
         
         return cell
     }
+
+	func helpLinkCell(text: String, url: URL, indexPath: IndexPath) -> UITableViewCell {
```
Comment (sche) [inline Multisig/UI/UI Library/UITableView+Cells.swift]: minor: not formatted here and below

**Review by sche — APPROVED** [review]

File: Multisig/UI/Settings/OwnerKeyManagement/OwnerKeyDetailsViewController/OwnerKeyDetailsViewController.swift [hunk 3]
```diff
@@ -172,7 +172,31 @@ class OwnerKeyDetailsViewController: UITableViewController {
...
+                 items: [Section.OwnerKeyType.type])]
+
+            if self.keyInfo.keyType == .walletConnect {
+                self.sections.append((section: .connected("WC CONNECTION"), items: [Section.Connected.connected]))
+            }
+
+            if [.walletConnect, .ledgerNanoX].contains(keyInfo.keyType) {
+                self.sections.append((section: .pushNotificationConfiguration("PUSH NOTIFICATIONS"),
+                                 items: [Section.PushNotificationConfiguration.enabled]))
+                if self.keyInfo.delegateAddress != nil {
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/OwnerKeyManagement/OwnerKeyDetailsViewController/OwnerKeyDetailsViewController.swift:192]: I think we'll need to re-do this part: we need to ask backend what delegate key exist on each network and display the list either here or in a separate screen... 

This is not what I intended, but it still works I guess for now.

Comment (MouazAlzahabi) [reply Multisig/UI/Settings/OwnerKeyManagement/OwnerKeyDetailsViewController/OwnerKeyDetailsViewController.swift:192]: [Ticket](https://app.zenhub.com/workspaces/mobile-development-board-60cc78740f488d000e996335/issues/gnosis/safe-ios/1680)

**Pull request merged**